src/content/docs/orgs.mdx

```diff
@@ -0,0 +1,168 @@
+same organization settings page.
+
+![Organization Settings with new
+Member](https://github.com/user-attachments/assets/4936b211-23d8-4a44-90e6-06fab421a4f4)
+
+## Managing Organization Apps
+
+Organizations can have multiple apps associated with them. You can either
+transfer existing apps to an organization or create new apps within the
+organization directly.
+
+To transfer an existing app to an organization, visit the app's settings and
+click the "Transfer app" button. You can then select the organization you want
+to transfer the app to from the dropdown.
+
+![Transfer
+App](https://github.com/user-attachments/assets/9d7c5428-ebfd-4c9c-9777-4ca06da8d962)
+
+To create a new app within an organization, select the organization when running
+`shorebird init`:
+
+![Shorebird Init with
+Organization](https://github.com/user-attachments/assets/6a07fb1f-1fc6-4b41-9df8-60a50c42b475)
+
+## Organization Roles
+
+Organization roles apply to all apps within the organization and can be managed
+via the organization settings page in the Shorebird console.
+
+If you have any questions about access control for your app, please do not
+hesitate to reach out to contact@shorebird.dev.
+
+### Organization Member
+
+New members added to an organization are assigned the "Member" role by default.
+
+Members can:
+
+- Read information for all apps within the organization
+- View collaborators on all apps within the organization
+- Create releases and patches for all apps within the organization
+- View insights for all apps within the organization
+
+### Organization Admin
+
+Organization Admins can:
+
+- Do everything a Member can do
+- Add and Remove Members for the organization
+- Delete Apps within the organization
+- Delete Releases within the organization
+
+### Organization Owner
+
+There is only one owner per organization (the person who created the
+organization) and they have full control over the organization and all apps
+within it.
+
+The owner can:
+
+- Do everything an Admin can do
+- Manage the billing information for the organization
+
+If you need more control, consider our [enterprise plan](https://shorebird.dev/pricing).
+
+## Managing Collaborators
+
+Per app collaborators can also be added and managed via the Shorebird console.
+To add a collaborator, from your app's page:
+
+1. Select the "Collaborators" tab.
+2. Click "Add people".
+3. Enter the email address of the person you would like to add as a
+   collaborator.
+   1. Note that the email address must be associated with an existing Shorebird
+      account.
+4. Click "Add Collaborator".
+
+Existing collaborators can be removed by clicking the trash icon next to their
+email address.
+
+## Collaborator Roles
+
+_Roles are per-app._ Each account can create Apps tied to that account. Once an
+App is created, access to that app can be shared. If you have any questions
+about access control for your app, please do not hesitate to reach out to
+contact@shorebird.dev.
```

Comment:
This is old.

Collaborators are used for per-app access controls.  Typically used to provide someone temporary access to only one of your applications.

Collaborators allow you to provide access to one of your apps without granting someone access to all of the apps in an organization.

Collaborators can be added both to apps within your personal storage as well as on any organization which you are an admin member of.

Collaborators can see the existence of an organization and its name, but cannot see any other aspects of an organization.